arches/app/views/search.py

```diff
@@ -260,7 +261,13 @@ def export_results(request):
         exporter = SearchResultsExporter(search_request=request)
         export_files, export_info = exporter.export(format, report_link)
         wb = export_files[0]["outputfile"]
-        with NamedTemporaryFile() as tmp:
+        
+        if sys.platform == "win32":
+            tmp = NamedTemporaryFile(dir=settings.TILE_EXCEL_EXPORT_TEMP_DIRECTORY, delete=settings.TILE_EXCEL_EXPORT_TEMP_FILE_DELETE)
+        else:
+            tmp = NamedTemporaryFile()
+                    
+        with tmp:
```

Comment:
What do you think about just making this platform-agnostic? This should work on all platforms, combined with an `os.unlink(tmp.name)` to get rid of the tempfile after the context manager exits.

```suggestion
        with NamedTemporaryFile(delete=False) as tmp:
```

Reply:
@jacobtylerwalls - I am thinking back a while here from when I first did the work on an internal system but I think I did try this and found that Windows (or maybe it was just our Keystone setup on Windows server) that required a specified directory for the temp file.  I'll give your suggestion a whirl on my local Windows setup to see if things will work as you've suggested